### DISCUSSION TITLE
Programming help with Flowchart Maker

### DISCUSSION THREAD

**Discussion by bluebouncyball** [community/community#63348] [Programming Help]
## Overview
I'm pretty new to programming, but I wanted to make a new program that is basically just a flowchart maker. I am currently mainly having a problem with figuring out how exactly I would go about creating working arrows with bends that do not go through objects. There is more info inside the repository I created, but I generally just need help sparking an idea on how to go about doing this, as well as some possible help with code efficiency.

## Code Repository
https://github.com/bluebouncyball/flowchart-maker

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by humairshoukat** [comment]
I thin this will help you https://github.com/cdfmlr/pyflowchart

**Reply by bluebouncyball** [reply]
As much I would love to get anything out of that program, I can not really understand how it at all works since I just started python, and it also does not seem to use Tkinter. Thank you for the suggestion, though.
